satellite/metainfo/validation.go

```diff
@@ -169,7 +176,7 @@ func (endpoint *Endpoint) validateCreateSegment(ctx context.Context, req *pb.Seg
 }
 
 func (endpoint *Endpoint) validateCommitSegment(ctx context.Context, req *pb.SegmentCommitRequest) (err error) {
-	defer mon.Task()(&ctx)(&err)
+	defer mon.Task()(&ctx)(nil)
```

Comment:
Just curious why are we explicitly passing `nil` here?

Reply:
Because of important reasons! No just a mispaste I guess. I'll fix!